### PR TITLE
Support for codeBlocks and inline code

### PR DESCRIPTION
**Pull request by imdfl**

### CODE REVIEW

File: src/components/content/content-blocks/block-quote/block-quote.st.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 @namespace "BlockQuote";
 
 .root {}
+
+.code {
+	white-space: pre;
```
Comment (tomerlichtash) [inline src/components/content/content-blocks/block-quote/block-quote.st.css:6]: Why do we need this style both in `BlockQuote` and in `CodeBlock`?

File: src/components/content/content-blocks/block-quote/block-quote.tsx [hunk 2]
```diff
@@ -7,9 +7,10 @@ import {
 import { st, classes } from "./block-quote.st.css";
 import { mlUtils } from "../../../../lib/ml-utils";
 
+
 export const BlockQuote = ({
 	componentData,
-	className,
+	className
```
Comment (tomerlichtash) [inline src/components/content/content-blocks/block-quote/block-quote.tsx:13]: Seems like unnecessary changes to this file.

File: src/components/content/content-blocks/code-block/code-block.st.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@namespace "CodeBlock";
+
+.root {
+	white-space: pre;
```
Comment (tomerlichtash) [inline src/components/content/content-blocks/code-block/code-block.st.css:4]: I tested the code block, and the styling has a bug - it cuts long lines.
```suggestion
	white-space: pre-wrap;
```
seems to solve this. Also, consider adding browser support for this:
```suggestion
	white-space: pre-wrap;       /* Since CSS 2.1 */
	white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
	white-space: -pre-wrap;      /* Opera 4-6 */
	white-space: -o-pre-wrap;    /* Opera 7 */
```

Sources:
- [StackOverflow](https://www.longren.io/wrapping-text-inside-pre-tags/)
- [Wrapping Text Inside Pre Tags](https://www.longren.io/wrapping-text-inside-pre-tags/)

**Review by tomerlichtash — CHANGES_REQUESTED** [review]
- Found one tiny bug and added a suggestion for a fix (`pre`)
- Added some minor _change requests_ for consistency sake.
- `mlType` FTW

File: src/components/content/content-blocks/block-quote/block-quote.tsx (hunk 2 above)
Comment (imdfl) [reply src/components/content/content-blocks/block-quote/block-quote.tsx:13]: leftovers from an attempted refactoring. If the change turns out to be detrimental to the project, we can roll it back in the next ML Hackathon.

File: src/components/content/content-blocks/block-quote/block-quote.st.css (hunk 1 above)
Comment (imdfl) [reply src/components/content/content-blocks/block-quote/block-quote.st.css]: Remains from a refactoring attempt. Reverted

File: src/components/content/content-component.tsx [hunk 4]
```diff
@@ -16,6 +16,7 @@ import { ContentIterator } from "./content-iterator";
 import CustomImage from "./content-blocks/custom-image";
 import LinkSelector from "./content-blocks/link-selector";
 import { st, classes } from "./content-component.st.css";
+import CodeBlock from "./content-blocks/code-block";
```
[the remark this answers is not in the document]
Comment (imdfl) [reply src/components/content/content-component.tsx]: Done

**Review by tomerlichtash — APPROVED** [review]
